portfolio/src/main/java/com/google/sps/servlets/LoginStatusServlet.java

```diff
@@ -28,16 +28,27 @@
  */
 @WebServlet("/login-status")
 public class LoginStatusServlet extends HttpServlet {
+  private static UserService userService;
+  private static String loginUrl;
+  private static String logoutUrl;
+
+  @Override
+  public void init() {
+    userService = UserServiceFactory.getUserService();
+    logoutUrl = userService.createLogoutURL("/");
```

Comment:
Shouldnt't these be something like "/login" and "/logout"?

It's unclear to me why this is using "/". I suspect this is due not having separate endpoints for login, logout links (you're just pointing to the page on which there's  a login/logout button).

A comment here would be great to convey the reasoning.

Reply:
createLogoutURL(destinationURL) returns a URL that can be used to display a logout link to the user and 
destinationURL is the URL where the users will be redirected after they log out.

The login and logout links are different, but indeed, the code is not clear, I will add more comments.